src/gui/optionsdialog.ui

```diff
@@ -231,6 +234,9 @@
                </item>
                <item>
                 <widget class="QCheckBox" name="checkAltRowColors">
+                 <property name="toolTip">
+                  <string>Every second row in table elements has a grey background (Except for search and log widgets)</string>
```

Comment:
This option only applies to the torrents table right? Then we can be specific about it and omit the part in parentheses.

Reply:
It's for transferlist, peerlist, trackerlist and filelist. I also forgot about RSS but i don't know if it has any tables.
https://github.com/qbittorrent/qBittorrent/blob/06ceac4cda7ac70700e08175b9622cf26a20fbeb/src/gui/mainwindow.cpp#L1462-L1465

Btw with theme support coming (and existing dark theme), "grey" doesn't fit here .

Reply:
Addressed both of these, take a look at the new extracomment and tooltip.

Reply:
"Every second row in table elements has a differently colored background"
could mean red then green then blue... sorry i tried to think of an explanation but the words "alternating row colors" keep popping in mind which is just repeating the option name.

Reply:
I don't want to remove it because the current comment seems even more confusing, but I'm not sure how to improve it without making it even harder to understand.

Reply:
In the latest patch, I removed the tooltip and updated the extracomment. When the time comes for themes, I believe the extracomment should be updated then.